Keas.Mvc/Models/KeyImport.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+public class KeyImport
+{
+    [Required]
+    public string Keynumber { get; set; }
+    [Required]
+    public string  SerialNumber { get; set; }
+    public string Description { get; set; }
+
+    public string KerbUser { get; set; }
+
+    public DateTime DateIssued { get; set; }
```

Comment:
@jcubbage Needs to be nullable and you need to validate when you read it. (HasValue, isn't in the future, dueDate isn't in the past, etc.)